Controller/PageAdminController.php

```diff
@@ -94,7 +94,9 @@ public function treeAction()
 
         $this->get('twig')->getExtension('form')->renderer->setTheme($formView, $this->admin->getFilterTheme());
 
-        return $this->render('SonataPageBundle:PageAdmin:tree.html.twig', array(
+        $templates = $this->retrieveAdminTemplates();
+
+        return $this->render($templates['tree'], array(
```

Comment:
Whit not using `$this->admin->getTemplate('tree')` directly instead of getting all admin templates?